templates/workspaces/base/terraform/keyvault.tf

```diff
@@ -136,11 +136,18 @@ resource "azurerm_key_vault_secret" "client_id" {
   ]
 }
 
+
+data "azurerm_key_vault_secret" "client_secret" {
+  count        = var.client_secret == "REDACTED" ? 1 : 0
```

Comment:
Perhaps we should extract the redacted value into a local so that it won't be hard coded twice?